src/Server_TEST.cc

```diff
@@ -1172,6 +1172,32 @@ TEST_P(ServerFixture, ResolveResourcePaths)
         "include_nested", "model.sdf"), true);
 }
 
+/////////////////////////////////////////////////
+TEST_P(ServerFixture, Stop)
+{
+  // Start server
+  ServerConfig serverConfig;
+  serverConfig.SetUpdateRate(10000);
+  serverConfig.SetSdfFile(std::string(PROJECT_SOURCE_PATH) +
```

Comment:
`joinpaths`

Reply:
Done in [40c37d2](https://github.com/gazebosim/gz-sim/pull/1551/commits/40c37d295c74c67bbcc2bbe7472a584f38a64a00)